cranelift/codegen/meta/src/shared/instructions.rs

```diff
@@ -3845,7 +3845,14 @@ pub(crate) fn define(
         .operands_out(vec![x]),
     );
 
-    let x = &Operand::new("x", Float);
+    let FloatScalar = &TypeVar::new(
+        "sFloat",
```

Comment:
```suggestion
        "FloatScalar",
```

Reply:
Oh .. about this. That was my instinct as well, but I didn't want to do sFloat because s in front is already used to designate signed. Such as sInt. I think FloatScalar is very clear. sFloat isn't.